### PR TITLE
german erekir & v7 translation

### PR DESCRIPTION
**Pull request by Catchears**
Original PR text below.
Is ready to merge whenever Erekir is ready for translations, but will be kept up to date until then.

Original text:
Cleans up missing translation. Leaving as draft for now as Erekir is not done and missing some bundle entries (e.g. the tutorial) and because there are some things I'm not quite sure about yet.

Ich habe ein Paar der neuen Übersetzungen (von @end-me-please) geändert, damit die anderen Namen besser passen, das wären:

- Gelber Stein -> Gelbstein
- Roter Stein -> Rotstein
- Berylliumhaltiger Stein -> Berylliumstein
- Rotes Eis -> Roteis

Diese Übersetzungen sind zwar vielleicht nicht ganz richtig, aber es sollte sich jeder denken können, was gemeint ist, und sie passen bei Sachen wie "Gelbsteinschlot" meiner Meinung nach einfach besser.

Mir fehlen jedoch Ideen für viele der neuen Einheiten und Geschösse:
block.breach.name = Breach
block.sublimate.name = Sublimate
block.diffuse.name = Diffuse
unit.vanquish.name = Vanquish
unit.conquer.name = Conquer
unit.avert.name = Avert
unit.disrupt.name = Disrupt
unit.evoke.name = Evoke
unit.incite.name = Incite
unit.emanate.name = Emanate
unit.elude.name = Elude
unit.obviate.name = Obviate

Einfach die Infinitivformen im Deutschen zu nehmen ergibt keine vernünftigen Namen (Durchbrechen, Stören). Fällt vielleicht einem anderen Übersetzer etwas ein?

### CODE REVIEW

**Comment by end-me-please** [comment]
(Was a bit on the fence for the rock names, thanks)

**Comment by Catchears** [comment]
Brecher klingt schonmal gut, Sublimat klingt mir noch zusehr nach einem Gegenstand als ein Geschütz, aber bis jetzt fällt mir da wenig anderes ein.

**Comment by CreepTV** [comment]
What about "Diffusor"?

**Comment by stacktrace-error** [comment]
The fact that most of the names are verbs makes this rather difficult as words like "Verbreiten" don't have any ring to them at all

**Comment by Catchears** [comment]
I've committed a first draft, I think the names sound alright and you can generally guess where I got the idea from. Does anyone mind these or have any suggestions? I'll mention @Shawak as I'm not sure you've seen this PR yet and would like to at least check with you.

**Comment by Catchears** [comment]
I've merged the modified changes from #6899 into this PR - they fix several missed translations.

File: core/assets/bundles/bundle_de.properties [hunk 1]
```diff
@@ -1563,8 +1563,8 @@ block.phase-wall.description = Beschützt Blöcke vor Gegnern, indem sie die mei
 block.phase-wall-large.description = Beschützt Blöcke vor Gegnern, indem sie die meisten Schüsse reflektiert.
 block.surge-wall.description = Beschützt Blöcke vor Gegnern und greift Gegner mit Lichtbögen an.
 block.surge-wall-large.description = Beschützt Blöcke vor Gegnern und greift Gegner mit Lichtbögen an.
-block.door.description = Eine Wand, die geöffnet und geschlossen werden kann.
-block.door-large.description = Eine Wand, die geöffnet und geschlossen werden kann.
+block.door.description = Eine Mauer, die geöffnet und geschlossen werden kann.
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:1566]: Ein Tor

File: core/assets/bundles/bundle_de.properties [hunk 2]
```diff
@@ -1563,8 +1563,8 @@ block.phase-wall.description = Beschützt Blöcke vor Gegnern, indem sie die mei
...
+block.door-large.description = Eine Mauer, die geöffnet und geschlossen werden kann.
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:1567]: Ein Tor

File: core/assets/bundles/bundle_de.properties [hunk 3]
```diff
@@ -1314,7 +1314,7 @@ block.duo.name = Doppelgeschütz
 block.scorch.name = Scatter
 block.scatter.name = Luftgeschütz
 block.hail.name = Hagel
-block.lancer.name = Lancier
+block.lancer.name = Lancer
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:1317]: Lanze?

File: core/assets/bundles/bundle_de.properties [hunk 4]
```diff
@@ -1954,90 +1954,90 @@ block.blast-door.name = Panzertür
...
-block.core-citadel.name = Core Citadel
-block.core-acropolis.name = Core Acropolis
+block.armored-duct.name = Gepanzerte Rohrleitung
+block.overflow-duct.name = Überlaufrohrleitung
+block.duct-unloader.name = Rohrleitungsentlader
+block.surge-conveyor.name = Spannungsförderband
+block.surge-router.name = Spannungsverteilung
+block.unit-cargo-loader.name = Einheitenlader
+block.unit-cargo-unload-point.name = Einheitentladungspunkt
+block.reinforced-pump.name = Verstärkte Pump
```
Comment (Shawak) [inline core/assets/bundles/bundle_de.properties:1968]: Pumpe

**Review by Shawak — CHANGES_REQUESTED** [review]

**Review by Shawak — APPROVED** [review]

**Comment by end-me-please** [comment]
oh crap, forgot about this draft before creating a new pr, thought it was already merged

https://github.com/Anuken/Mindustry/pull/6999
oh well

**Comment by Catchears** [comment]
I can merge it in here if you want - then we'd have the changes combined.

**Comment by Catchears** [comment]
I'm going to mark this as ready-for-review.

I will use this opportunity to offer to merge #6870 and #6999 (both by @end-me-please) into this PR to avoid merge conflicts through multiple PRs, just drop a comment below if you'd like me to.

**Pull request merged**